packages/themes/src/index.ts

```diff
@@ -35,12 +72,7 @@ export const themeIds = [
   'none',
```

Comment:
this doesn't seem to do much and is breaking theme when used, we might want to remove it?

Reply:
Yeah @hwkr @hanspagel is there a reason for this? I just left it as is from before.

Reply:
This is used to override the default option for the references but it probably only actually needs to be in the refs